src/bitcoincore.rs

```diff
@@ -164,6 +164,10 @@ impl BitcoinRpcApi for BitcoinCoreClient {
         ))
     }
 
+    fn get_balance(&self) -> Result<Result<f32, RpcError>, HTTPError> {
```

Comment:
According to the API here: https://bitcoin.org/en/developer-reference#getbalance, `getbalance` has 3 parameters:

- an account (although this one is deprecated, I think we should still add it for now) it is an optional parameter, so we should probably accept an `Option<String>` and default to `*` as described in the docs if it is missing.
- minimum number of confirmations
- whether to include watch-only addresses

Please add these 3 parameters to the method.